README.md

```diff
@@ -12,20 +12,49 @@
 <a href="https://smartthingsdev.slack.com/messages/CG595N08N"><img src="https://badgen.net/badge//smartthingsdev?icon=slack"/></a>
 </p>
 
-#### [Reference Documentation](docs/index.md)
+## Reference Documentation
 
-SDK that wraps the SmartThings REST API and reduces the amount of code necessary to write a SmartApp app. It supports 
-both webhook and AWS Lambda implementations. This is a preview version of the API and will change over time time.
+Looking for reference documentation? Find it [here](docs/index.md).
 
-## Version 2.0 Release
+## Getting Started
 
-_ATTENTION! This major release is not completely backwardly compatible with the 1.X version, though for most
-SmartApps the changes required should be relatively minor. The major non-backwardly changes are:_
-* _Methods that return lists now return arrays rather that an object with the properties `items` and `_links`._
-* _Axios is now used rather than request-promise-native for making HTTP calls, resulting in changes to the error
+>**Version 2.0 Release**
+>
+>This is a preview version of the API and will change over time.
+>
+>*This major release is not completely backwardly compatible with  version 1.X*, though for most SmartApps the changes required should be relatively minor. The major non-backwards compatible changes include:
+>* _Methods that return lists now return arrays rather that an object with the properties `items` and `_links`._
+>* _Axios is now used rather than request-promise-native for making HTTP calls, resulting in changes to the error
 objects thrown when exceptions occur._
+>
+>See the [Version 2.0.0 release notes](docs/V2_RELEASE_NOTES.md) for more information.
 
-_See the [Version 2.0.0 release notes](docs/V2_RELEASE_NOTES.md) for more information._
+This SDK includes a set of NodeJS libraries for building Webhook and AWS Lambda SmartApps, and interacting with the public SmartThings API.
```

Comment:
A nit, but `Node.js` is a more popular way of writing it.

Reply:
Revised, thanks for catching that.